keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,426 @@
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies — for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+#### Feature graduation
+
+Research discovered that only 5% of Kubernetes features advanced from Alpha to
+GA in the minimum 3 releases. However, the same research showed that reminders
+from the Release Team played a critical role in advancement of more than 50% of
+features. With a longer release cycle, this reminder activity can be
+expected to slow down. As such, advancement will need to be mitigated by making
+sure that SIGs keep track of their feature enhancement in more detail.
+
+#### Unprepared Kubernetes Users
+
+Kubernetes effectively moved to a *3 release per year* cadence in 2020, starting 
+with the Kubernetes 1.19 release. At the start of the Kubernetes 1.21 release, 
+there was communication that a permanent cadence change was under consideration, 
+however this KEP was not submitted and approved within the Kubernetes 1.21 release 
+cycle, so there is a risk that downstream consumers may be unaware that such a 
+change was under consideration and could be caught by surprise by a cadence change. 
+Some downstream projects, such as Helm have already begun 
+[planning](https://github.com/helm/community/blob/main/hips/hip-0002.md#minor-releases) 
+for this change.
+
+To mitigate this risk, SIG-Release will perform the following actions:
+
+* Once this KEP has merged, an email will be sent to the [k/dev](https://groups.google.com/g/kubernetes-dev) list
+* A community meeting occurring during the 1.22 Release Cycle will be used to communicate the change
+* Early in the Kubernetes 1.22 Release cycle, a blog will be written and published to 
+https://kubernetes.io/blog/ that fully explains this change.
+* A tweet (linking to the blog) will sent from the k8scontributors twitter account
+
```

Comment:
Risk: Increased risk-taking in enhancement lifecycle.  On the 4-release cadence, a relatively trivial enhancement could move from alpha to beta to GA in 9 months, with truly trivial features sometimes skipping beta.  On the proposed 3-release cadence, The best possible case is 12 months for a 3-phase features and 8 months if skipping beta.  These drawn out timelines may cause more features to skip beta or take more risks in advancing phases, even when not confident.  The cost of "missing the boat" goes up.  The only mitigation here is human vigilance and engineering discipline to hold the line and say "no" when appropriate.  This is VERY difficult and not to be trivialized.

Reply:
Added the risk to the KEP as suggested.